Extensions/anti_capitalism.js

```diff
@@ -61,13 +61,17 @@ XKit.extensions.anti_capitalism = new Object({
 		if (XKit.page.react) {
 			await XKit.css_map.getCssMap();
 
+			//exit if we're on a grid page to avoid breaking anything
+			if ($(XKit.css_map.keyToCss("masonry")).length) { return; }
```

Comment:
i'm not confident that this is the best solution - this means the extension would disable itself if you hard-load a masonry tagged page (or search page!), including when the user soft-switches back to the dashboard. can this logic be moved to the function that erroneously hides content on masonry tagged pages?

(also, nittiest of nitpicks, but i think "grid" should only refer to the plain tile-like view for likes and the other tagged page; if "masonry" is an unclear term, maybe "bricks view" would be a good replacement?)

Reply:
Ah, yeah, soft refreshes. I think I deleted my branch with a really complicated css selector that fulfilled this requirement; lemme go dig it up :D 

And whoa I had no idea what you guys were talking about, but it looks like I have the non-masonry tagged page now, so I see what you mean. I kinda like just sticking to "masonry."